.circleci/test.yml

```diff
@@ -146,7 +146,7 @@ workflows:
           name: minimum_version_cpu
           torch: 1.6.0
           torchvision: 0.7.0
-          python: 3.6.9  # The lowest python 3.6.x version available on CircleCI images
+          python: 3.7.4
```

Comment:
Why this mini version of 3.7

Reply:
we are going to deprecate the support of python3.6 and the opencv doesn't support python3.6 now. Thus modifying this and not run unit test on python3.6 in the future.

Reply:
My question is why is the specific version of python 3.7.4, instead of other versions of python 3.7.x

Reply:
> My question is why is the specific version of python 3.7.4, instead of other versions of python 3.7.x

just copied from mmcv pr https://github.com/open-mmlab/mmcv/pull/2525